scripts/linters/custom_eslint_checks/rules/no-multiline-disable.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Are we expecting to have test for this check? [What should we do to ensure the coverage is 100% or at least we have test files for each rules, any idea?]

Reply:
>  [What should we do to ensure the coverage is 100% or at least we have test files for each rules, any idea?]

Any thoughts on this?

Reply:
We are using mocha for tests, we can use nyc with mocha to get the codecoverage.

Reply:
Create an issue to add test coverage for the eslint tests*

Reply:
Opened the issue https://github.com/oppia/oppia/issues/10552